.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
Ideally this should be in your global .gitignore on your local machine, but it’s not something I feel strongly about.